src/services/ai-server/ai-persona-service/dto/ai.persona.service.invocation.dto.input.ts

```
@@ -57,7 +58,7 @@ export class ResultHandler {
     message:
       'roomDetails with roomID, threadID and communicationID is required when action is POST_REPLY',
   })
-  validateRoomDetails() {
+  validateRoomDetails?() {
     return this.roomDetails && this.roomDetails.threadID;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)